src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -65,11 +65,18 @@ public function supports(string $resourceClass, string $operationName = null): b
      *
      * @throws RuntimeException
      */
-    public function getItem(string $resourceClass, $id, string $operationName = null, array $context = [])
+    public function getItem(string $resourceClass, $id, string $operationName = null, array $context = []/** , array $identifiers **/)
```

Comment:
This was the only solution I had to avoid breaking the `DataProvider::getItem` signature. Also, in future versions `$identifier` should replace `$id` and it will be typed as `array`.